src/components/Quotes.js

```diff
@@ -1,39 +1,13 @@
-import { useState, useEffect } from 'react';
-
 function Quotes() {
-  const [data, setData] = useState([]);
-
-  useEffect(() => {
-    const fetchData = async () => {
-      const res = await fetch('https://api.api-ninjas.com/v1/quotes?limit=10&category=money', {
-        headers: {
-          'X-Api-Key': '417MZ7DAIIDohifAq1HR0g==C8ex0UAh8uWfaqqG',
-        },
-      });
-      const json = await res.json();
-      setData(json);
-    };
-    fetchData();
-  }, [setData]);
-
   return (
-    <ul>
-      {data.map((item) => (
-        <li key={item.id}>
-          <strong>Author:</strong>
-          {' '}
-          {item.author}
-          {' '}
-          <strong>Category:</strong>
-          {' '}
-          {item.category}
-          {' '}
-          <strong>Quote:</strong>
-          {' '}
-          {item.quote}
-        </li>
-      ))}
-    </ul>
+    <>
+      <p style={{
+        fontSize: 20, color: 'black', paddingTop: '10vw', paddingLeft: '10vw',
+      }}
+      >
+        Pure mathematics is, in its way, the poetry of logical ideas.— Albert Einstein.
+      </p>
+    </>
```

Comment:
> - [ ] Hello @alex1779, I've noticed that you removed the previous code responsible for fetching quotes from the API. Presently, there's static text in its place, which doesn't meet the project's requirements. In the previous review, a suggestion was made to enhance the appearance and readability of the quotes obtained from the API, rather than completely removing the existing code. I kindly request that you reimplement the Quote API functionality as previously done and consider enhancing the presentation by increasing the font size and center-aligning it. By making these adjustments, you'll successfully complete the Math Magician project.👍